### PR TITLE
Upgrade to Angular 8

### PR DESCRIPTION
**Pull request by marjan-georgiev**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

Supports Angular 8

**Does this PR introduce a breaking change?** (check one with "x")
- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/bar-chart/bar.component.ts [hunk 1]
```diff
@@ -37,7 +36,7 @@ import { DataItem } from '../models/chart-data.model';
 })
 export class BarComponent implements OnChanges {
   @Input() fill;
-  @Input() data: DataItem;
+  @Input() data: any;
```
Comment (duredhel13) [inline src/bar-chart/bar.component.ts]: What happened here? I'm curious.

Comment (marjan-georgiev) [reply src/bar-chart/bar.component.ts]: Rollup complaining about DataItem not being exported. It is.

**Comment by vdumbrav** [comment]
@marjan-georgiev , https://github.com/swimlane/ngx-charts/issues/1202

**Comment by victorvilardo** [comment]
angular/platform-browser-dynamic  / platform-browser - 8.2.14 incompatibility

**Pull request merged**